### PR TITLE
torch.special.airy_ai

### PR DESCRIPTION
**Pull request by 0x00b1**
```Python
airy_ai(input, *, out=None) -> Tensor
```

Airy function $\text{Ai}\left(\text{input}\right)$.

### CODE REVIEW

File: torch/special/__init__.py [hunk 1]
```diff
@@ -871,6 +872,20 @@
 
 """.format(**common_args))
 
+airy_ai = _add_docstr(_special.special_airy_ai,
+                      r"""
+airy_ai(input, *, out=None) -> Tensor
+
+Airy function :math:`\text{Ai}\left(\text{input}\right)`.
```
Comment (mruberry) [inline torch/special/__init__.py:879]: I'm not sure how much the latex helps explain what this function does, but OK -- we should probably elaborate on a lot of these special functions with more mathematical formulas later

File: torch/testing/_internal/common_methods_invocations.py [hunk 2]
```diff
@@ -19093,6 +19093,27 @@ def error_inputs_mean(op_info, device, **kwargs):
...
+                {
+                    torch.float32: 1e-03,
+                    torch.float64: 1e-05,
+                },
+            ),
+        ),
+        dtypes=all_types_and(torch.bool),
+        ref=lambda x: scipy.special.airy(x)[0] if TEST_SCIPY else _NOTHING,
+        skips=(
+            DecorateInfo(
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:19109]: Comment for this skip, please

**Review by mruberry — APPROVED** [review]
Looks good except for those ONNX and Buck job failures we should figure out

**Review by linbinyu — APPROVED** [review]

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**